patches/chrome-BUILD.gn.patch

```diff
@@ -60,9 +60,12 @@ index 9bb34fe40f0cc08cb832ee6728ac8e32458c969b..d9996a8931038ceebb89b77db255f675
        }
  
        extra_substitutions = [
-@@ -915,6 +922,7 @@ if (is_win) {
+@@ -913,8 +920,9 @@ if (is_win) {
+     }
+ 
      if (enable_updater) {
-       if (is_chrome_branded) {
+-      if (is_chrome_branded) {
++      if (true) {
          sources += [ "//third_party/updater/chrome_mac_universal_prod/cipd/${updater_product_full_name}.app" ]
 +      } else if (use_prebuilt_omaha4) { sources += [ "//build/mac_files/omaha4/${updater_product_full_name}.app" ]
```

Comment:
this branch will never be hit. intentional?

Reply:
Good catch, thank you. Fixed in a7b5990e4395f37d4089ead5105e49257b5d9160.